[2021-01-21 23:20:31,677] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_2 2021-01-21T23:20:22.188540+00:00 [queued]>
[2021-01-21 23:20:31,694] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_2 2021-01-21T23:20:22.188540+00:00 [queued]>
[2021-01-21 23:20:31,694] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-01-21 23:20:31,694] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-01-21 23:20:31,694] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-01-21 23:20:31,703] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): task_2> on 2021-01-21T23:20:22.188540+00:00
[2021-01-21 23:20:31,706] {standard_task_runner.py:51} INFO - Started process 576671 to run task
[2021-01-21 23:20:31,719] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_2', '2021-01-21T23:20:22.188540+00:00', '--job-id', '97', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmpfn3cfi64']
[2021-01-21 23:20:31,719] {standard_task_runner.py:76} INFO - Job 97: Subtask task_2
[2021-01-21 23:20:31,772] {logging_mixin.py:103} INFO - Running <TaskInstance: trigger_rule.task_2 2021-01-21T23:20:22.188540+00:00 [running]> on host ip-172-31-18-6.ap-south-1.compute.internal
[2021-01-21 23:20:31,824] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-01-21T23:20:22.188540+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-01-21T23:20:22.188540+00:00
[2021-01-21 23:20:31,824] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-01-21 23:20:31,825] {bash.py:158} INFO - Running command: exit 0
[2021-01-21 23:20:31,833] {bash.py:169} INFO - Output:
[2021-01-21 23:20:31,836] {bash.py:177} INFO - Command exited with return code 0
[2021-01-21 23:20:31,853] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_2, execution_date=20210121T232022, start_date=20210121T232031, end_date=20210121T232031
[2021-01-21 23:20:31,895] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-01-21 23:20:31,930] {local_task_job.py:118} INFO - Task exited with return code 0
